package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cls-mongoose-shim",
-  "version": "0.0.8",
+  "version": "1.0.0",
```

Comment:
bumped major version as this would be a breaking change moving from mongoose v4 or v5 to v6